README.md

```diff
@@ -9,36 +9,43 @@ dependencies for use by the charm.
 # Usage
 
 To create a charm layer using this base layer, you need only include it in
-a ``layer.yaml`` file:
+a `layer.yaml` file:
 
-```yaml
+``yaml
```

Comment:
Shoot, I got a little trigger-happy with the global-search-and-replace.  Thanks.